team.py

```diff
@@ -52,46 +65,124 @@ def edit_password(self, teamname, input):
     def answer_question(self, teamname, input):
         # teamname: string
         # input: list of length 1
+        time_check = self.check_time_up()
+        current_time = datetime.now(tz=timezone.utc)
         answer = input[0]
         game = Game.objects.get(name="game")
         if not game.running:
+            if time_check:
+                return "Time is up!"
             return "There is no game running!"
         try:
             team = HuntUser.objects.get(name=teamname)
         except HuntUser.DoesNotExist:
             return "Not a valid team!"
+        if team.game_ended:
+            return "You've already finished the game!"
+        if team.current_landmark.name == "dummy":
+            return "Not at a valid landmark"
+        if not team.question_requested:
+            return "You need to receive the question first!"
         if answer != team.current_landmark.answer:
+            team.penalties += 5
```

Comment:
Should be an additional check to see how many times they've answered wrong, with that check being changeable by the gamemaker.